fixtures/dom/package.json

```
@@ -1,3 +1,5 @@
+Making a change
```

dsfkjngvjkdsfnjsdkjnfkjsdnfdsf

```suggestion
don't make this change 
```